### PR TITLE
Level creator

### PR DESCRIPTION
**Pull request by n0remac**
This branch has some new movement code in it. To use the original blessed threaded movement use the --bless tag when starting app. otherwise it will use the pynput movement. 

This branch also added new levels when you go through a door.

### CODE REVIEW

File: src/resources/GameResources.py [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
 class GameResources:
     """Holds objects that are used for during game runtime"""
 
-    def __init__(self, testing: bool):
-        self.level = Level(20, 15, [1, 2, 3, 4], [])
+    def __init__(self, testing: bool, bless: bool):
+        self.level_selector = LevelSelector()
+
+        self.level = self.level_selector.create_level()
         self.player = Character(symbol="$", x=self.level.width // 2, y=self.level.height // 2)
```
Comment (Antsthebul) [inline src/resources/GameResources.py:15]: When a level is created can we ensure that there will be no enemey RIGHT in front of the player when a new one is created?

**Review by Antsthebul — APPROVED** [review]
Also, this does not pertain to this PR specifically but i noticed the commands are being saved to lists. What is this for?

File: src/resources/GameResources.py (hunk 1 above)
Comment (n0remac) [reply src/resources/GameResources.py:15]: New enemies are no created yet. It just leaves the enemies from the last level in place. We sill need to talk about how enemy spawning will work in the final product.

**Comment by n0remac** [comment]
Command are stored in a queue by the controller on the separate thread. The command are then popped from the main thread once per tick to be used.

**Review by cons01e — APPROVED** [review]

**Pull request merged**